stacker/providers/aws/default.py

```diff
@@ -614,11 +614,17 @@ def tail(self, stack_name, cancel, log_func=_tail_print, sleep_time=5,
                 return
 
     def destroy_stack(self, stack, **kwargs):
+        from pprint import pprint
```

Comment:
Don't see this used... perhaps you were using it to debug?

Reply:
Yup. I added pre-commit to prevent that from happening again.